spec/unit/details.polyfill.spec.js

```diff
@@ -0,0 +1,91 @@
+/* global describe it expect beforeEach afterEach */
+
+var $ = window.jQuery
+
+describe('details-polyfill', function () {
+  'use strict'
+  var GOVUK = window.GOVUK
+
+  beforeEach(function (done) {
+    // Sample markup
+    this.$content = $(
+      '<details>' +
+      '<summary><span class="summary">Summary</span></summary>' +
+      '<div><p>Hidden content</p></div>' +
+      '</details>'
+    )
+
+    // Find elements
+    var $summaries = this.$content.find('summary')
+    var $hiddenContent = this.$content.find('div')
+
+    this.$summary1 = $summaries.eq(0)
+    this.$hiddenContent1 = $hiddenContent.eq(0)
+
+    // Add to page
+    $(document.body).append(this.$content)
+
+    setTimeout(function () {
+      done()
+    }, 1)
+  })
+
+  afterEach(function () {
+    this.detailsPolyfill = null
+    this.$content.remove()
+  })
+
+  describe('and when details.polyfill is initialised', function () {
```

Comment:
Suggest just "When the polyfill is initialised" – there's no real prior statement to connect the 'and' to.